### PR TITLE
Add CHANGELOG.md for TouchFree and Web Tooling

### PR DESCRIPTION
**Pull request by Craig-J**
## Summary

_Summary of the purpose of this pull request._

Add a changelog to track changes for both:

- TouchFree - documents changes to all components released together in the main bundle with the touchfree installer
- TouchFree Web Tooling - documents changes to the web tooling which is released separately to the above

All historical releases added to the changelogs already.

### Motivation

Intent is to enhance our workflow when dealing with the internal release notes page on confluence.
Developers will manage software changes and document them in the changelog, which in turn can be refined into product release notes on the release info pages.
Changelogs are intended to replace the release notes field on Jira tickets which have not been getting used consistently.

- With a changelog in the repo we can review it closely with contributions to the code itself during PRs.
- Changes for a release are built up naturally as we merge rather than requiring collation from release notes when performing the release process.
- CHANGELOG.md is a common practice by adopted by many projects. This makes it naturally discoverable.
- CHANGELOG.md can be distributed directly with the release, allowing changes to be viewed offline.

### CODE REVIEW

File: TF_Tooling_Web/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+# Changelog
+All notable changes to the TouchFree Web Tooling project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+[docs-website]: https://docs.ultraleap.com/touchfree-user-manual/ "Ultraleap Docs"
```
Comment (JordanBerni-UL) [inline TF_Tooling_Web/CHANGELOG.md]: 👀 This line doesn't appear when viewing the page

Comment (JordanBerni-UL) [reply TF_Tooling_Web/CHANGELOG.md]: image

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+- System Tray: The Windows tray icon
+- Overlay Application: The TouchFree Overlay Application launched from the tray icon that enables using TouchFree as a pointing device
+
+TouchFree web tooling changes are documented separately.
+See [here if looking at the source](./TF_Tooling_Web/CHANGELOG.md) otherwise [here on GitHub](https://github.com/ultraleap/TouchFree/blob/develop/TF_Tooling_Web/CHANGELOG.md).
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+[docs-website]: https://docs.ultraleap.com/touchfree-user-manual/ "Ultraleap Docs"
```
Comment (JordanBerni-UL) [inline CHANGELOG.md]: 👀 Line missing (maybe because the space in the URL?)

File: TF_Tooling_Web/CHANGELOG.md (hunk 1 above)
Comment (Craig-J) [reply TF_Tooling_Web/CHANGELOG.md]: Replaced with some badges at the top (with discord and support email too)

File: CHANGELOG.md (hunk 2 above)
Comment (Craig-J) [reply CHANGELOG.md]: Replaced with some badges at the top (with discord and support email too)

**Review by JordanBerni-UL — APPROVED** [review]

**Review by JordanBerni-UL — CHANGES_REQUESTED** [review]
Removing approval as want to add changelog for brightsign

**Review by JordanBerni-UL — COMMENTED** [review]
Windows changelog is missing https://github.com/ultraleap/TouchFree/pull/410

BRS changelog is missing https://github.com/ultraleap/TouchFree/pull/412

File: .github/PULL_REQUEST_TEMPLATE/lightweight.md [hunk 3]
```diff
@@ -19,6 +19,7 @@ _Summary of the purpose of this pull request._
 _These tasks are for the pull request creator to tick off._
 
 - [ ] Developer testing
+- [ ] Make sure relevant changelog is updated if any changes affect users
```
Comment (JordanBerni-UL) [inline .github/PULL_REQUEST_TEMPLATE/lightweight.md]: Can you link to the changelogs in the PR description to make it more convenient? And can you move Developer Testing to the Reviewer Tasks here? (in all PR templates)

Comment (Craig-J) [reply .github/PULL_REQUEST_TEMPLATE/lightweight.md]: Added. Can't test PR template relative links are correct without merge 😞

**Comment by Craig-J** [comment]
> Windows changelog is missing #410
> 
> BRS changelog is missing #412

Added #410 to changes to web tooling changelog. Nothing visible for users of main product yet so not mentioned in touchfree windows changelog.

Added #412 to brightsign changelog.

**Review by JordanBerni-UL — APPROVED** [review]

**Pull request merged**